app/views/layouts/application.html.erb

```diff
@@ -34,6 +34,11 @@
     <meta property="twitter:card" content="summary_large_image">
     <meta property="twitter:url" content="<%= canonical_url %>">
 
+    <% # Site verification for InsightAppSec DAST vulnerability scanner %>
+    <% if not Rails.env.production? %>
```

Comment:
```suggestion
    <% unless Rails.env.production? %>
```
so we don't actually follow any [ruby style guides](https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#if-vs-unless) _but_ we also don't have any instances of `if not` and tend to use `unless` around the codebase